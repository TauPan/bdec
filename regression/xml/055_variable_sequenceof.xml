<protocol>
    <sequence name="blah">
        <sequence name="a">
            <field name="a1" length="8" type="integer" />
        </sequence>
        <sequenceof name="b" count="${a.a1}">
            <sequence name="b1">
                <field name="b2" length="8" type="integer" />
            </sequence>
        </sequenceof>
    </sequence>

</protocol>
