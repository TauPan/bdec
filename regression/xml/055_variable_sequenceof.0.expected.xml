
           <blah>
              <a>
                 <a1>3</a1>
              </a>
              <b>
                 <b1><b2>0</b2></b1>
                 <b1><b2>0</b2></b1>
                 <b1><b2>83</b2></b1>
              </b>
           </blah> 